README.md

```diff
@@ -30,3 +30,158 @@ If you have any questions or ideas feel free to join our <a href="https://discor
+
+    <template>
+    <DxpOmsInstanceNavigator />
+    </template>
+
+    <script>
+    import { DxpOmsInstanceNavigator } from "@hotwax/dxp-components";
+    </script>
+
+###  Events 
+#### 1. goToOms()
+<li>The purpose of goToOms is to navigate the user to a different screen or page within the application that corresponds to a specific OMS instance.
+<li>It takes two parameters: <b>token</b> (an authentication token) and <b>omsName</b> (the name of the OMS instance).
+
+
+### state manamement 
+#### 1.useAuthStore()
+ It simplifies access to authentication state.
+ The state in useAuthStore manages the data related to authentication, which is token and oms.
+
+
+## DxpUserProfile  
+### Introduction
+<li>The User Profile Card component is designed to display user profile information, allowing users to see details such as their login ID and party name. It also provides options to perform actions like logging out and navigating to the launchpad.
+
+### Basic Usage
+UserProfile component provides a user-friendly display of profile information.
+
+To use the DxpUserProfile component, integrate it into your Vue 3 application.
+
+      <template>
+      <DxpUserProfile @reset-state-before-logout="resetStateBeforeLogout" :userProfile="userProfile" logoutLabel="Logout" goToLabel="Go to Launchpad" />
+      </template>
+
+      <script>
+      import { DxpUserProfile } from "@hotwax/dxp-components";
+      </script>
+    
+
+### Props
+<table>
+    <thead>
+      <tr>
+        <th>Prop Name</th>
+        <th>Description</th>
+        <th>Type</th>
+        <th>Required</th>
+        <th>Default Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>userProfile</code></td>
+        <td>User profile information object</td>
+        <td>Object</td>
+        <td>Yes</td>
+        <td>-</td>
+      </tr>
+      <tr>
+        <td><code>logoutLabel</code></td>
+        <td>Label for the logout button</td>
+        <td>String</td>
+        <td>No</td>
+        <td>'Logout'</td>
+      </tr>
+      <tr>
+        <td><code>goToLabel</code></td>
+        <td>Label for the go-to launchpad button</td>
+        <td>String</td>
+        <td>No</td>
+        <td>'Go To Launchpad'</td>
+      </tr>
+    </tbody>
+  </table>
+
+### Events
+#### 1. logout()
+The logout() method is triggered when the user clicks on the logout button (ion-button). It initiates the logout process for the user.
+##### Events inside logout
+<li><b>emit()</b> : The emit function is used to emit the before-logout event. Emitting the 'before-logout' event allows parent components or listeners to be aware of the impending logout action.
+<li><b>dispatch()</b> : It dispatches the logout action defined in the user module and redirects to the login page.
+
+#### 2. goToLaunchpad()
+The goToLaunchpad() method is invoked when the user clicks on the "Go To Launchpad" button. It redirects the user to the launchpad by changing the window location.
+
+### CSS
```

Comment:
I don't think we need to really go this deep into the code in this draft. Lets focus on the functionality and usage first.